Content.IntegrationTests/Tests/Networking/SimplePredictReconcileTest.cs

```diff
@@ -136,24 +136,27 @@ public async Task Test()
 
                     // Event got repeated on client as a past prediction.
                     Assert.That(clientSystem.EventTriggerList,
```

Comment:
Somehow got here as False, True, True, True. Fix.

Reply:
Might be fixed by dirtying the comp after handling state.

Reply:
Did not fix it. Not sure what's wrong here.

Reply:
Looks like the component state isn't being reset during prediction? I don't know why that's happening.